stetho/src/main/java/com/facebook/stetho/inspector/elements/android/TextViewDescriptor.java

```diff
@@ -71,5 +95,49 @@ public void afterTextChanged(Editable s) {
         getHost().onAttributeModified(mElement, TEXT_ATTRIBUTE_NAME, s.toString());
       }
     }
+
+    void setAttributeAsText(String text) {
+      // Text will be in the format "attribute1=\"Value 1\" attribute2=\"Value2\""
```

Comment:
this parsing and splitting and mapping stuff should be in a utility method somewhere so that we can easily use it from other descriptors too. For now it's probably sufficient to put it as a static protected method in Descriptor.